bcgov/biohubbc
# Fix app login redirect bug

# Overview

Fixes a bug that prevents users from being correctly redirected to the login page

## Reproducing the Bug
0. Go to SIMS dev, or run SIMS locally on the dev branch
1. Sign in to SIMS.
2. Once signed in, change your path to `/login`. For example, `localhost:7100/login`, or `https://sims-dev-url.gov.bc.ca/login`
3. You should:

 - [x] See a blank white screen.

## Testing the hotfix
0. Checkout the hotfix branch, or visit the PR app route at: https://biohubbc-app-1031-af2668-dev.apps.silver.devops.gov.bc.ca/
1. Sign in to SIMS.
2. Once signed in, change your path to `/login'.
3. You should:

- [x] Be redirected to the SIMS home page.

4. Next, change your path to `/login?redirect=%2Fadmin%2Fprojects`
5. You should:

 - [x] Be redirected to the SIMS projects list page.
